SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -146,7 +146,13 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	void UpdateSpatialPositionWithFrequencyCheck();
 	void UpdateSpatialPosition();
 
-	void ServerProcessOwnershipChange();
+private:
+	FString CachedOwnerWorkerAttribute;
```

Comment:
Think this should be next to FString SavedOwnerWorkerAttribute; in the existing private section. Also add a comment as to what this is used for since we are tracking quite a few of these WorkerAtributes